docs/components/search/index.html

```diff
@@ -24,64 +24,98 @@ <h2 id="basic-search">Basic Search</h2>
-            <input
-              id="txtValue"
-              type="text"
-              v-model="searchValue"
-            />
-            <label for="txtValue">
-              Value
-            </label>
-          </hx-text-control>
+          <fieldset>
+            <legend class="beta-hxFieldName">Label Annotation</legend>
+            <hx-checkbox-control>
+              <input
+                id="chkHasAsterisk"
+                type="checkbox"
+                v-model="hasAsterisk"
+              />
+              <label for="chkHasAsterisk">
+                <hx-checkbox></hx-checkbox>
+                Required
+              </label>
+            </hx-checkbox-control>
+
+            <hx-checkbox-control>
+              <input
+                id="chkHasOptional"
+                type="checkbox"
+                v-model="hasOptional"
+              />
+              <label for="chkHasOptional">
+                <hx-checkbox></hx-checkbox>
+                Optional
+              </label>
+            </hx-checkbox-control>
+          </fieldset>
 
           <fieldset>
-            <legend class="beta-hxFieldName">Options</legend>
+            <legend class="beta-hxFieldName">Control Options</legend>
             <hx-checkbox-control>
               <input
-                id="chkIsDisabled"
+                id="chkDisabled"
                 type="checkbox"
                 v-model="isDisabled"
               />
-              <label for="chkIsDisabled">
+              <label for="chkDisabled">
                 <hx-checkbox></hx-checkbox>
                 Disabled
               </label>
             </hx-checkbox-control>
 
             <hx-checkbox-control>
               <input
-                id="chkIsInvalid"
+                id="chkRequired"
                 type="checkbox"
-                v-model="isInvalid"
+                v-model="isRequired"
               />
-              <label for="chkIsInvalid">
+              <label for="chkRequired">
                 <hx-checkbox></hx-checkbox>
-                Invalid
+                Required
               </label>
+              <p>
+                (Invalid styling applied <i>after</i> user interaction.)
+              </p>
             </hx-checkbox-control>
           </fieldset>
         </form>
       </header>
 
       <div>
-        <hx-search
-          :disabled="isDisabled"
-          :invalid="isInvalid"
-          :placeholder="placeholder"
-          :value="searchValue"
-          @clear="onSearchUpdate"
-          @input="onSearchUpdate"
-        ></hx-search>
+        <hx-search-control>
+          <input
+            id="srchInputDemo"
+            type="search"
+            :disabled="isDisabled"
+            :required="isRequired"
+          />
+          <button
+            type="button"
+            class="hxClear"
+            aria-label="Clear search"
+            hidden
+          >
+            <hx-icon type="times"></hx-icon>
+          </button>
```

Comment:
Phase II: ... hide button if `<hx-search-control>` _hasn't_ upgraded.

```css
hx-search-control:not([hx-defined]) > button {
  display: none;
}
```

Reply:
Added above styles in PR #620.